Makefile

```
@@ -3,6 +3,7 @@ COVERAGE := --cov=pennylane_lightning --cov-report term-missing --cov-report=htm
 TESTRUNNER := -m pytest tests --tb=short
 
 PL_BACKEND ?= "$(if $(backend:-=),$(backend),lightning_qubit)"
+SCIPY_OPENBLAS :=$(shell $(PYTHON) -c "import scipy_openblas32; import os; print(os.path.dirname(scipy_openblas32.get_lib_dir()))")
```

I'm a bit confused! Didn't we want the path to `scipy_openblas32/lib` in C++ src? If so, why do we need to pass the directory name then? We should be able to do this:  
```suggestion
SCIPY_OPENBLAS_LIB :=$(shell $(PYTHON) -c "import scipy_openblas32; print(scipy_openblas32.get_lib_dir())")
```

---

Nice catch! Thanks !!